### ISSUE TITLE
krun/kompile couldn't find temp files and throws java.io.FileNotFoundException

### ISSUE THREAD

**Issue by yilongli** [kframework/k-legacy#1092]
I have observed this problem several times on my machine.

Here is one I just got from Jenkins:

```
java.io.FileNotFoundException: ./.krun-2014-10-22-15-30-48-1d1b113b-b7bf-471c-aeb6-33d4434468e1/timing.log (No such file or directory)
    at java.io.FileOutputStream.open(Native Method)
    at java.io.FileOutputStream.<init>(FileOutputStream.java:206)
    at java.io.FileWriter.<init>(FileWriter.java:107)
    at org.kframework.parser.generator.ParseRulesFilter.visit(ParseRulesFilter.java:103)
    at org.kframework.parser.generator.ParseRulesFilter.visit(ParseRulesFilter.java:31)
    at org.kframework.kil.StringSentence.accept(StringSentence.java:36)
    at org.kframework.kil.AbstractVisitor.visitNode(AbstractVisitor.java:84)
    at org.kframework.kil.AbstractVisitor.genericVisitList(AbstractVisitor.java:135)
    at org.kframework.kil.AbstractVisitor.visit(AbstractVisitor.java:300)
    at org.kframework.parser.generator.ParseRulesFilter.visit(ParseRulesFilter.java:50)
    at org.kframework.parser.generator.ParseRulesFilter.visit(ParseRulesFilter.java:31)
    at org.kframework.kil.Module.accept(Module.java:111)
    at org.kframework.kil.AbstractVisitor.visitNode(AbstractVisitor.java:84)
    at org.kframework.kil.AbstractVisitor.genericVisitList(AbstractVisitor.java:135)
    at org.kframework.kil.AbstractVisitor.visit(AbstractVisitor.java:284)
    at org.kframework.kil.Definition.accept(Definition.java:166)
    at org.kframework.kil.AbstractVisitor.visitNode(AbstractVisitor.java:92)
    at org.kframework.parser.DefinitionLoader.parseString(DefinitionLoader.java:340)
    at org.kframework.krun.tools.Prover$Tool.run(Prover.java:73)
    at org.kframework.krun.tools.Prover$Tool.run(Prover.java:39)
    at org.kframework.transformation.TransformationCompositionProvider$2.run(TransformationCompositionProvider.java:120)
    at org.kframework.krun.KRunFrontEnd.run(KRunFrontEnd.java:70)
    at org.kframework.main.FrontEnd.main(FrontEnd.java:43)
    at org.kframework.main.Main.main(Main.java:40)
```

http://office.runtimeverification.com:8080/rv-jenkins/job/k-framework-pull-requests-maven/org.kframework.k$k-distribution/351/testReport/junit/(root)/_samples_kernelc/copy/

This issue is not related to the failure of the test, but it's annoying.

@osa1 I think this is related to your recent changes.

**Comment by radumereuta** [comment]
As a side note, 'timing.log' is a file that I create if you use the -v option, and it contains a list of how much time it takes to parse each rule. Because parsing might fail at any point, I open and close this file each time I parse a sentence.

**Comment by osa1** [comment]
I think this is related with Dwight's last files API update.

**Comment by dwightguth** [comment]
Fixed 97f9284
